### PR TITLE
Update comparison_to_alternatives.mdx

### PR DESCRIPTION
**Pull request by gmourier**
# Pull Request

## What does this PR do?

- Add the search analytics capabilities in the comparative table.

### CODE REVIEW

File: learn/what_is_meilisearch/comparison_to_alternatives.mdx [hunk 1]
```diff
@@ -111,6 +111,7 @@ Can't find a client you'd like us to support? [Submit your idea or vote for it](
 |   | Meilisearch | Algolia | Typesense | Elasticsearch |
 |---|:---:|:----:|:---:|:---:|
 | [Mini Dashboard](https://github.com/meilisearch/mini-dashboard) | ✅ | 🔶 <br /> Cloud product | 🔶 <br /> Cloud product | ✅ |
+| Search Analytics | ✅ <br/> Cloud product | ✅ <br /> Cloud Product | ❌ | ✅ <br /> Cloud Product |
```
[the remark this answers is not in the document]
Comment (Strift) [reply learn/what_is_meilisearch/comparison_to_alternatives.mdx]: I think it's the best category for now

[the remark this answers is not in the document]
Comment (Strift) [reply learn/what_is_meilisearch/comparison_to_alternatives.mdx]: I'm not against adding a link to our LP if we have one too!

**Comment by Strift** [comment]
LGTM — you can add the link to the LP if we have one, but it's good as is too.

File: learn/what_is_meilisearch/comparison_to_alternatives.mdx [hunk 2]
```diff
@@ -111,6 +111,7 @@ Can't find a client you'd like us to support? [Submit your idea or vote for it](
 |   | Meilisearch | Algolia | Typesense | Elasticsearch |
 |---|:---:|:----:|:---:|:---:|
 | [Mini Dashboard](https://github.com/meilisearch/mini-dashboard) | ✅ | 🔶 <br /> Cloud product | 🔶 <br /> Cloud product | ✅ |
+| Search Analytics | ✅ <br /> Cloud product | ✅ <br /> Cloud Product | ❌ | ✅ <br /> Cloud Product |
```
Comment (gmourier) [inline learn/what_is_meilisearch/comparison_to_alternatives.mdx:114]:
```suggestion
| Search Analytics | ✅ <br /> [Cloud product](https://www.meilisearch.com/cloud) | ✅ <br /> Cloud Product | ❌ | ✅ <br /> Cloud Product |
```

**Comment by gmourier** [comment]
@Strift, I've put the /cloud LP link instead since we don't have the SA LP ready yet. Please let me know if you wanna insert some utm tracking !

**Review by guimachiavelli — APPROVED** [review]
LGTM

Should I merge it or should we wait for @Strift's final ok?

**Comment by gmourier** [comment]
We can go @guimachiavelli ! Thanks

**Pull request merged**